src/hooks/screen/useChatScreenHook.ts

```diff
@@ -239,6 +254,7 @@ function useChatScreenHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
     sendChat,
     handleUserName,
     updateChatContinuity,
+    selfAnonUserInfo,
     flatListRef,
     scrollContext: {
       selectedMessageId,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

> Note: This review was outside of the patch, so it was mapped to the patch with the greatest overlap. Original lines [257-263]

The `selfAnonUserInfo` state variable is now included in the returned object of the hook. Make sure all components using this hook are updated to handle this new property correctly.
```diff
+ selfAnonUserInfo,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->